File: csharp/beecrowd1009.cs
using System;
using System.Globalization;

class Programa
{
    static void Main(string[] args)
    {
        CultureInfo CI = CultureInfo.InvariantCulture;

        string nome;
        double salario, totalVendas, totalReceber;

        nome = Console.ReadLine();
        salario = double.Parse(Console.ReadLine(), CI);
        totalVendas = double.Parse(Console.ReadLine(), CI);

        totalReceber = salario + totalVendas * 0.15;

        Console.WriteLine("TOTAL = R$ " + totalReceber.ToString("F2", CI));
    }
}

File: cpp/beecrowd1009.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {

    string nome;
    double salario, totalVendas, totalReceber;

    cin >> nome >> salario >> totalVendas;

    totalReceber = salario + totalVendas * 0.15;

    cout << fixed << setprecision(2);
    cout << "TOTAL = R$ " << totalReceber << endl;

    return 0;
}
